e2e/utils.go

```diff
@@ -1353,6 +1353,7 @@ func retryKubectlFile(namespace string, action kubectlAction, filename string, t
 // retryKubectlArgs takes a namespace and action telling kubectl what to do
 // with the passed arguments. This function retries until no error occurred, or
 // the timeout passed.
+// nolint:unparam // retryKubectlArgs will be used with kubectlDelete arg later on.
```

Comment:
should `deleteResource` use this function then?

Reply:
its using `retryKubectlInput`  
https://github.com/ceph/ceph-csi/blob/90d246fc55142fbcfe9ef8fd34c4a20526c00ce0/e2e/utils.go#L126